core/admin/mailu/models.py

```diff
@@ -334,6 +335,46 @@ def check_mx(self):
         except dns.exception.DNSException:
             return False
 
+    def check_spf(self):
+        """ checks if SPF record for domain points to mailu host """
+        try:
+            hostnames = app.config['HOSTNAMES'].replace(',', '|')
+            return any(
+                re.search(f'^"v=spf1 mx a:{hostnames} [?+-~]all"$', rset.to_text())
```

Comment:
I think hostnames should be grouped: `f'^"v=spf1 mx a:({hostnames}) [?+-~]all"$'`

Reply:
We can improve all regexes if there is expert about dns records. Yeah my mistake, i will update this.

Reply:
Done